lib/apiservers/engine/backends/eventmonitor.go

```diff
@@ -198,19 +210,26 @@ func (m *PortlayerEventMonitor) monitor() error {
 }
 
 // publishEvent() translate a portlayer event into a Docker event if the event is for a
-// known container and publishes them to Docker event subscribers.  Currently, it only
-// looks for stopped events.
+// known container and publishes them to Docker event subscribers.
 func (p DockerEventPublisher) PublishEvent(event plevents.BaseEvent) {
 	defer trace.End(trace.Begin(""))
 
 	vc := cache.ContainerCache().GetContainer(event.Ref)
 	if vc == nil {
 		log.Errorf("Portlayer event for container %s but not found in cache", event.Ref)
 	}
+	var dockerEvent string
+
+	switch event.Event {
+	case plevents.ContainerStopped:
+		dockerEvent = containerDieEvent
+	case plevents.ContainerStarted:
+		dockerEvent = containerStartEvent
```

Comment:
So, I should have brought this up before.  If we were to follow Docker's pattern, their events are generally the result of a request fullfilled and not the operation completing.  In other words, with the exception of 1 or 2 events (e.g. die), all their events are published in the routers at the end of their related function.  So for start, that would get published at the end of a successful ContainerStart.  Also, not all events will have a corresponding event from the portlayer.  I think we should move this event to ContainerStart.